go.mod

```diff
@@ -199,6 +199,7 @@ require (
 	github.com/containerd/stargz-snapshotter/estargz v0.14.3 // indirect
 	github.com/containerd/ttrpc v1.1.0 // indirect
 	github.com/containerd/typeurl v1.0.2 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
```

Comment:
You also need to run `go mod tidy`.

Reply:
go.sum will be updated

Reply:
Yes, i pushed the updated go.sum file too

Reply:
Thanks